docs/release-notes.md

```diff
@@ -1,3 +1,19 @@
+## Release v1.2.0
+### Date: November 30, 2023
+
+This release focused on increasing the security of PASS and making interactions with external services more robust, notably the interface with NIHMS. Parameterized queries were added to the grant loader to enhance security. We've made substantial upgrades to the NIHMS data transfer within our Deposit Services and enhancements to the NIHMS loader. Updates were made to the data model documentation. A new JHU logo has been introduced and client-side pagination support has been added in the UI.
```

Comment:
Need to check on the inclusion of the JHU logo in the codebase.

Reply:
Verified with Jared. This should not be in the release-notes as this is JHU specific. I have removed JHU logo language.